WordPress/Classes/Services/JetpackNotificationMigrationService.swift

```diff
@@ -38,6 +40,8 @@ final class JetpackNotificationMigrationService: JetpackNotificationMigrationSer
         }
 
         set {
+            userDefaults.set(true, forKey: wordPressNotificationsToggledDefaultsKey)
```

Comment:
It returns an optional (which can be cast to a bool) so I think it can capture the three values we're looking to store here:
- user hasn't changed their notification preference
- user has turned off notifications 
- user has manually turned on notifications